seacrowd/sea_datasets/xnli/xnli.py

```diff
@@ -0,0 +1,218 @@
+  location = "Brussels, Belgium",
+}
+"""
+
+_DATASETNAME = "xnli"
+
+_DESCRIPTION = """\
+XNLI is a subset of a few thousand examples from MNLI which has been translated into a 14 different languages (some low-ish resource). As with MNLI, the goal is to predict textual entailment (does sentence A imply/contradict/neither sentence B) and is a classification task (given two sentences, predict one of three labels).
+"""
+
+_HOMEPAGE = "https://github.com/facebookresearch/XNLI"
+
+# We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LANGUAGES = ["tha", "vie"]
+_LANGUAGE_MAPPER = {"tha": "th", "vie": "vi"}
+
+_LICENSE = Licenses.CC_BY_NC_4_0.value
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: {
+        "train": "https://dl.fbaipublicfiles.com/XNLI/XNLI-MT-1.0.zip",
+        "test": "https://dl.fbaipublicfiles.com/XNLI/XNLI-1.0.zip",
+    }
+}
+
+_SUPPORTED_TASKS = [Tasks.TEXTUAL_ENTAILMENT]
+
+_SOURCE_VERSION = "1.1.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class XNLI(datasets.GeneratorBasedBuilder):
+    """
+    XNLI is an evaluation corpus for language transfer and cross-lingual sentence classification in 15 languages.
+    In SeaCrowd, we currently only have Thailand and Vietnam Language.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    subsets = ["xnli.tha", "xnli.vie"]
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{sub}_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description=f"{sub} source schema",
+            schema="source",
+            subset_id=f"{sub}",
+        )
+        for sub in subsets
+    ] + [
+        SEACrowdConfig(
+            name=f"{sub}_seacrowd_pairs",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description=f"{sub} SEACrowd schema",
+            schema="seacrowd_pairs",
+            subset_id=f"{sub}",
+        )
+        for sub in subsets
+    ]
+
+    DEFAULT_CONFIG_NAME = "xnli.vie_source"
+    labels = ["contradiction", "entailment", "neutral"]
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "premise": datasets.Value("string"),
+                    "hypothesis": datasets.Value("string"),
+                    "label": datasets.ClassLabel(names=self.labels),
+                }
+            )
+
+        elif self.config.schema == "seacrowd_pairs":
+            features = schemas.pairs_features(self.labels)
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(
+        self, dl_manager: datasets.DownloadManager
+    ) -> List[datasets.SplitGenerator]:
```

Comment:
change on one-liner based on code-formatter
```suggestion
    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
```